src/lib/tutorial.js

```diff
@@ -50,4 +51,20 @@ export function display (t) {
     cozyBarMenuButton.click()
     window.location.hash = '#/discovery'
   })
+  if (isMobile.phone) {
+    const clickZone = '.introjs-disableInteraction, .introjs-overlay, .introjs-tooltiptext, .introjs-tooltipbuttons'
+    const clickAction = (e) => {
+      for (const elem of document.querySelectorAll(clickZone)) {
```

Comment:
I saw that introJs have a `nextStep()` method, did we try to keep a reference to the introJs object ?
We should do something like :
```javascript
const clickAction = (e) => {
  e.stopPropagation() // avoid conflicts
  introJs.nextStep()
}
```